contributors/devel/development.md

```diff
@@ -136,8 +136,8 @@ platform.
 
 ### jq
 
-Kube-apiserver requires the `jq` to be installed to successfully build your Local Kubernetes Deployment. 
-You can see installation guide for `jq` over [here](https://stedolan.github.io/jq/download/).
+Kube-apiserver requires `jq` to be installed to successfully build your local Kubernetes environment. 
```

Comment:
s/build your local Kubernetes environment/build Kubernetes in your local environment/

Reply:
done